paper_framework_sources/p1898_forward_progress_delegation.md

```diff
@@ -159,10 +166,12 @@ Therefore the most appropriate solution is to make available a valid downstream
 # Generalising
 Let us say that a given asynchronous algorithm promises some forward progress guarantee that depends on its executor/scheduler^[Potentially also on the execution property passed but we leave that out here for simplicity.].
 
+Note that in the examples below `Sender<T>` refers to a `sender` that will call `set_value` on a `receiver` passed to its `connect` operation with type `T`, and exists for exposition only.
```

Comment:
Consider renaming `Sender` to _`sender-of`_. The standard uses italics kebab case for exposition-only stuff, and _`sender-of`_`<int>` reads nicely.

Reply:
I think I actually removed that anyway based on your feedback on the other PR.